test/mri/tests/ruby/test_gc.rb

```diff
@@ -8,6 +8,20 @@ def initialize(a)
     end
   end
 
+  def test_gc_unknown_key
+    keys = [:omganyvalue, :total_allocated_objects]
+    keys.each do |key|
+      assert_equal(0, GC.stat(key))
+    end
+  end
+
+  def test_gc_known_key
+    keys = [:count, :time]
+    keys.each do |key|
+      assert GC.stat(key) > 0
+    end
+  end
+
```

Comment:
I don't think we've ever modified MRI tests like this before - we don't really have any mechanism to up-stream them and they may not even get accepted!

Would you be willing to implement these as specs instead? We can help you do that if you aren't sure what I mean.

Reply:
> Would you be willing to implement these as specs instead?

Absolutely. Should this go under `spec/ruby/core/gc/`?

Reply:
Yes :+1: